### PR TITLE
Add a Dockerfile and a docker image to test Docker containers RAM usage.

### PR DESCRIPTION
**Pull request by zoobab**
Add a Dockerfile and a docker image to test Docker containers RAM usage.

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+screen -d -m /bin/eatmemory ${RAMINMB}M
```
Comment (julman99) [inline entrypoint.sh]: why not just pass the arguments to the docker container? that way you can do this:

```
docker run -d  zoobab/eatmemory:latest 128M
docker run -d  zoobab/eatmemory:latest 5G
```

**Comment by zoobab** [comment]
Yeah it is simpler with just $1 as argument.

Now for the sleep, there is also "sleep infinity", but it does not seem to work on alpine.

**Comment by julman99** [comment]
Thanks a lot for your contribution!

**Comment by zoobab** [comment]
Actually the example is wrong in the README, if I pass 128MB as argument it does not work out, if I pass 128M it works. Can you confirm and fix master?

**Pull request merged**